modin/config/pubsub.py

```diff
@@ -135,11 +135,14 @@ class Parameter(object):
         ``ValueSource``.
     """
 
-    choices: typing.Sequence[str] = None
+    choices: Optional[Sequence[str]] = None
     type = str
-    default = None
+    default: Optional[Any] = None
     is_abstract = True
```

Comment:
does `is_abstract` need a type hint `bool`?

Reply:
It didn't need the type hint, I think it was able to figure it out on its own.